src/Core/Application/Application.csproj

```diff
@@ -15,4 +15,7 @@
         <ProjectReference Include="..\..\Core\Domain\Domain.csproj" />
         <ProjectReference Include="..\..\Shared\Shared.DTOs\Shared.DTOs.csproj" />
     </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\GlobalUsings.cs"/>
```

Comment:
Good idea to put it in a shared source file... hadn't thought of that ;-)

Maybe better to name it something like GlobalDefaultIdType.cs, or GlobalTypeDefs? So it's more clear and people won't start throwing in using statements willy nilly?

Reply:
yeah, naming to `GlobalTypeDefs` is a good choice
but I'm not sure who `Directory.Build.props` can be used.